client/post-editor/editor-page-parent/index.jsx

```diff
@@ -22,25 +24,36 @@ export default React.createClass( {
 	},
 
 	updatePageParent( item ) {
+		const parentId = item && item.ID ? item.ID : null;
 		postActions.edit( {
-			parent: item.ID
+			parent: parentId
 		} );
 	},
 
 	getEmptyMessage() {
 		if ( this.props.postId ) {
 			return this.translate( 'You have no other pages yet.' );
-		} else {
-			return this.translate( 'You have no pages yet.' );
 		}
+
+		return this.translate( 'You have no pages yet.' );
 	},
 
 	render() {
 		return (
 			<AccordionSection className="editor-page-parent">
-				<label>
+				<FormLabel>
 					<span className="editor-page-parent__label-text">{ this.translate( 'Parent Page' ) }</span>
```

Comment:
Unrelated to this pull request, but seems like we could move the `className` to the `<FormLabel>` component and have it work just as well.